cfgov/v1/atomic_elements/organisms.py

```diff
@@ -678,3 +678,36 @@ class Meta:
         label = 'Chart Block'
         icon = 'image'
         template = '_includes/organisms/chart.html'
+
+
+class SnippetList(blocks.StructBlock):
+    heading = blocks.CharBlock(required=False)
+    body = blocks.RichTextBlock(required=False)
+    image = atoms.ImageBasic(required=False)
+
+    snippet_type = blocks.ChoiceBlock(
+        choices=ref.snippet_types,
```

Comment:
Using a callable here would avoid the generation of migrations should the list of snippets change in future. See related comment below about potentially moving away from a hardcoded list.